build-tools/api-merge/api-merge.cs

```diff
@@ -77,6 +92,55 @@ public static int Main (string[] args)
 			return 0;
 		}
 
+		static int RunConfigurationFile (string config, string inputDir, string outputDir)
+		{
+			if (!File.Exists (config)) {
+				Console.WriteLine ($"error: config file {config} not found");
+				return 3;
+			}
+
+			var doc = XDocument.Load (config);
+			var inputs = doc.Root.Element ("Inputs").Elements ("File").Select (elem => Tuple.Create (Path.Combine (inputDir, elem.Attribute ("Path").Value), elem.Attribute ("Level").Value)).ToList ();
```

Comment:
Could we please *avoid* `Tuple`?  `ValueTuple<...>` would be fine, so long as the members have name.  Reading `.Item1` makes my eyes bleed.  Or just add a new `struct`!

Reply:
Nice, didn't realize named tuples were that easy!